http2-grpc-proto3-wire/http2-grpc-proto3-wire.cabal

```diff
@@ -1,10 +1,10 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.33.0.
+-- This file has been generated from package.yaml by hpack version 0.35.0.
```

Comment:
Is this change related to the ghc bump?

Reply:
Stack @ 2.9.1 bumps hpack to 0.35, and to my understanding this is the version supported by lts-20.4, see https://www.stackage.org/lts-20.4. Is this change problematic?